components/x-live-blog-wrapper/storybook/index.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withKnobs, text } from '@storybook/addon-knobs';
 import { LiveBlogWrapper } from '../src/LiveBlogWrapper';
+import '../../x-live-blog-post/dist/LiveBlogPost.css';
```

Comment:
How was the styling of the component in Storybook working without this?

Reply:
It wasn't working in the x-live-blog-wrapper storyboard.